tmdb/parsers/shows.py

```diff
@@ -70,6 +71,17 @@ def _get_next_episode_date(self, data: dict) -> Union[datetime.date, None]:
         next_episode: dict = data['next_episode_to_air'] or {}
         return self._parse_date(next_episode.get('air_date'))
 
+    def _get_list_seasons(self, data: dict) -> List[Season]:
+        seasons = data['list_seasons']
+        list_seasons = []
+        for season_number in range(len(seasons)):
+            season_parser = SeasonParser()
```

Comment:
Same suggestion here about reusing the `SeasonParser` and perhaps even condensing everything using a list comprehension.